### DISCUSSION TITLE
Feature Request or Assistance: Background Customization Based on Default Theme

### DISCUSSION THREAD

**Discussion by JefNewTech** [Clooos/Bubble-Card#739] [Questions about config, custom styles and templates]
Hello,

I would like to customize the background color of the Bubble Card based on the default (light or dark) theme in Home Assistant. Currently, I haven't been able to implement this functionality with the available options. Could you please provide a working example or consider adding this feature natively to the card?

Thank you in advance for your help!

**Comment by JefNewTech** [comment]
I am trying to achieve this with the following function:

.bubble-icon {
  color: 
    ${hass.states['[[entity]]'].state.includes('PMC') ? (hass.themes['current-theme'] === 'dark' ? 'white' : 'blue') :
      hass.states['[[entity]]'].state.includes('Ordures ménagères') ? (hass.themes['current-theme'] === 'dark' ? 'white' : 'gray') :
      hass.states['[[entity]]'].state.includes('Papier') ? (hass.themes['current-theme'] === 'dark' ? 'white' : 'brown') :
      (hass.themes['current-theme'] === 'dark' ? 'white' : 'black')} !important;
  font-size: px !important;
}

**Comment by Clooos** [comment]
Hi, the best way would be to define new variables in your theme file and using them in your custom styles.

**Reply by JefNewTech** [reply]
The aim is to use the bottom theme of the home assistant so as not to create a new one.

**Reply by Clooos** [reply]
In that case you can use the default variable, like var(--primary-text-color), this one is white on the dark theme and black on the light one.

**Reply by JefNewTech** [reply]
The best thing would be to create a file with the basic colors of home assistant and customize it so that I can do whatever I want with the bubble-cards. 
Do you have any idea where I can find the basic colors or the theme file for home assistant? I've searched their code but can't find it.

**Reply by Clooos** [reply]
I made some research and in fact you can just create a new theme with only the variables that you want to change, the other styles will stay by default.

**Reply by JefNewTech** [reply]
thanks for your help i've created a theme file which is much simpler for color management. 
Can you tell me if it is possible to create popup but with decluterring-card because mine remain transparent.

**Reply by Clooos** [reply]
I have tried in the past and it's not working with it 😕

**Reply by JefNewTech** [reply]
Thanks for your help
